src/mimefactory.rs

```diff
@@ -79,9 +79,9 @@ impl<'a, 'b> MimeFactory<'a, 'b> {
         let mut recipients = Vec::with_capacity(5);
         let mut req_mdn = false;
 
-        if chat.is_self_talk() {
+        if chat.is_self_talk() || context.get_config_bool(Config::BccSelf) {
```

Comment:
Doesn't it result in our own address being added to the "To:" field?

Reply:
Ok, this is not used for the "To:" field.

Reply:
more concrete, i think in a multi-device-setup, this results in the `To:` header _only_ containing our own address - and none of the other members in a group. while this does not affect the messages being sent, the recipients have no chance to reply to the group as the member list is not transmitted.